runtime/src/lib.rs

```diff
@@ -26,8 +26,12 @@ const fn is_testnet() -> bool {
     !env!("CARGO_PKG_VERSION_PRE").is_empty()
 }
 
-/// Determine EVM chain ID to use depending on whether the build is for Testnet or Mainnet.
+/// Determine EVM chain ID to use depending on whether the build is for Localnet, Testnet or Mainnet.
 const fn chain_id() -> u64 {
+    if option_env!("OASIS_UNSAFE_USE_LOCALNET_CHAINID").is_some() {
+        // Localnet.
+        0x5afd
```

Comment:
```suggestion
        0x5b00
```
or `0xf1b5` "[fibs](https://www.merriam-webster.com/dictionary/fib)"

Reply:
I'm not sure 🤔 How do we enumerate Emerald's (and any future runtime) localnet chain ID then? My current form was `localnet_chaind = min(mainnet_chainid, testnet_chainid)-1`.